backend/BCAPIServer/src/main/java/routes/bookclubs/UnjoinClub.java

```diff
@@ -0,0 +1,52 @@
+package routes.bookclubs;
+
+import com.google.firebase.FirebaseApp;
+import com.google.gson.JsonObject;
+import daos.User;
+import daos.UserResult;
+import java.sql.Connection;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class UnjoinClub implements Route {
+
+    private FirebaseApp fbApp;
+    private Connection sqlConn;
+
+    public UnjoinClub(FirebaseApp fbApp, Connection sqlConn) {
+        this.fbApp = fbApp;
+        this.sqlConn = sqlConn;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
+        JsonObject respJson = new JsonObject();
+
+        String user = request.queryParams("userId");
```

Comment:
Instead of taking in the user Id, this the route should take in the user's supplied authentication token and then use Firebase API to resolve the user Id from the user's authentication token. (see `LikePost` lines 31 to 51 for reference)

Reply:
In that case the URL in postman to test would be 
http://localhost:4567/api/join_club?token=auth_token&book_key=OL27448W

auth tokens expire and refresh so the way I get one without having to code is make a post in the web app, and then --> inspect --> application --> session storage --> http//localhost:3000
![image](https://user-images.githubusercontent.com/34294313/221688202-e59f1104-b2a6-49c0-ae61-8d647b11955e.png)